testdata/p4_16_errors_outputs/stack3.p4-stderr

```diff
@@ -0,0 +1,36 @@
+../testdata/p4_16_errors/stack3.p4(34): error: header h[]: Size of header stack type should be a constant
+        h[s1.size + 1] s2;
+        ^^^^^^^^^^^^^^
+../testdata/p4_16_errors/stack3.p4(34): error: header h[]: Size of header stack type should be a constant
+        h[s1.size + 1] s2;
+        ^^^^^^^^^^^^^^
+../testdata/p4_16_errors/stack3.p4(34): error: header h[]: Size of header stack type should be a constant
+        h[s1.size + 1] s2;
+        ^^^^^^^^^^^^^^
+../testdata/p4_16_errors/stack3.p4(34): error: Could not find type of h[]
+        h[s1.size + 1] s2;
+        ^^^^^^^^^^^^^^
+../testdata/p4_16_errors/stack3.p4(35): error: header h[]: Size of header stack type should be a constant
+        h[x] s3;
+        ^^^^
+../testdata/p4_16_errors/stack3.p4(35): error: header h[]: Size of header stack type should be a constant
+        h[x] s3;
+        ^^^^
+../testdata/p4_16_errors/stack3.p4(35): error: header h[]: Size of header stack type should be a constant
+        h[x] s3;
+        ^^^^
+../testdata/p4_16_errors/stack3.p4(35): error: header h[]: Size of header stack type should be a constant
+        h[x] s3;
+        ^^^^
+../testdata/p4_16_errors/stack3.p4(35): error: header h[]: Size of header stack type should be a constant
+        h[x] s3;
+        ^^^^
+../testdata/p4_16_errors/stack3.p4(35): error: header h[]: Size of header stack type should be a constant
+        h[x] s3;
+        ^^^^
+../testdata/p4_16_errors/stack3.p4(35): error: header h[]: Size of header stack type should be a constant
+        h[x] s3;
+        ^^^^
+../testdata/p4_16_errors/stack3.p4(35): error: Could not find type of h[]
```

Comment:
No, if a sub-expression fails type-checking no type is installed in the typeMap, so all the expressions that involve it as a sub-expression will fail to type-check too. These additional error messages are somewhat redundant, but other compilers are equally confused.